packages/cloud_firestore/cloud_firestore/test/transaction_test.dart

```diff
@@ -2,14 +2,13 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-// @dart=2.9
-
 // import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:firebase_core/firebase_core.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 import './mock.dart';
 
+// ignore: todo
```

Comment:
Why is this needed?

Reply:
I wasn't sure if it was my place to delete as it addresses @Ehesp. The comment specifically says remove when null safety lands.

Reply:
Let's just remove this file, it's doing nothing

Reply:
To add to this though I don't like that `ignore: todo` has been added to every todo, CI doesn't fail on todos and this ignore also isn't handled by IntelliJ IDEs so seems pointless - if you want to hide these in vscode then there's a setting on vscode to do just that:

![image](https://user-images.githubusercontent.com/5347038/106477371-0072ca80-64a0-11eb-8f91-f75aa795a964.png)